### PR TITLE
Introduce `Database` persistence

### PR DESCRIPTION
**Pull request by ZaiZu2**
Dependency to:
- https://github.com/ForloopAI/forloop_platform/pull/887

### CODE REVIEW

File: dbhydra/src/abstract_db.py [hunk 1]
```diff
@@ -105,6 +105,7 @@ def __init__(self, config_file="config.ini", db_details=None):
         
         self.active_transactions=[]
         self.last_table_inserted_into: Optional[str] = None
+        self.identifier_quote = ''
```
Comment (DovaX) [inline dbhydra/src/abstract_db.py:108]: isn't this empty string?

File: dbhydra/src/tables.py [hunk 2]
```diff
@@ -725,12 +725,12 @@ def get_num_of_records(self):
         Returns the number of records in table
         """
 
-        num_of_records = self.select(f"SELECT COUNT(*) FROM {self.name};")
+        num_of_records = self.select(f"SELECT COUNT(*) FROM `{self.name}`;")
```
Comment (DovaX) [inline dbhydra/src/tables.py:728]: shouldnt here be "{quote}"?

**Review by DovaX — APPROVED** [review]

File: dbhydra/src/abstract_db.py (hunk 1 above)
Comment (ZaiZu2) [reply dbhydra/src/abstract_db.py:108]: Yes, specified in the AbstractDb so it does not break concrete classes which do not have their own `self.identifier_quote`  but rely on methods inherited from `AbstractDb` where `self.identifier_quote` is used.

File: dbhydra/src/tables.py (hunk 2 above)
Comment (ZaiZu2) [reply dbhydra/src/tables.py:728]: Could be `self.db1.identifier_quote` but in concrete classes I was using specific quotes to decrease verbosity.

**Pull request merged**